CHANGELOG/CHANGELOG-1.4.md

```diff
@@ -0,0 +1,85 @@
+# Changelog 1.4
+
+## 1.4.0 - 2021-11-03
+
+The NodePortLocal feature is graduated from Alpha to Beta.
+
+### Added
+
+- Support for proxying all Service traffic, including NodePort, LoadBalancer, and ClusterIP traffic. Therefore, running kube-proxy is no longer required. ([#2599](https://github.com/antrea-io/antrea/pull/2599) [#2235](https://github.com/antrea-io/antrea/pull/2235) [#2897](https://github.com/antrea-io/antrea/pull/2897) [#2863](https://github.com/antrea-io/antrea/pull/2863), [@hongliangl] [@lzhecheng])
+  * The feature works for both Linux and Windows
+  * The feature is experimental and therefore disabled by default. Use the `antreaProxy.proxyAll` configuration parameter for the Antrea Agent to enable it
+  * If kube-proxy is removed, the `kubeAPIServerOverride` configuration parameter for the Antrea Agent must be set to access apiserver directly.
+- Add [AntreaIPAM] feature to allow flexible control over Pod IP Addressing. ([#2956](https://github.com/antrea-io/antrea/pull/2956), [@gran-vmv] [@annakhm])
+  * Add new IPPool API to define ranges of IP addresses which can be used as Pod IPs
+  * A Pod's IP will be allocated from the IPPool specified by the `ipam.antrea.io/ippools` annotation of the Pod's Namespace if there is one
```

Comment:
I haven't been following the feature quite closely, but per-IPAM Namespace is the only case we support right now? If yes I think the main CHANGELOG entry should reflect that:

> Add AntreaIPAM feature to allow flexible control over Pod IP addressing by assigning pools of IP addresses to specific Namespaces

(I thought we also supported Deployments, StatefulSets and possibly annotating individual Pods directly)